File: Java/Miscellaneous/Country.java
package LTD_API;

public class Country
{
    private int countryId;
    private String countryName;
    private int ordering;
    private String iso3166;
    
    public int getCountryId()
    {
        return countryId;
    }
    public void setCountryId(int countryId)
    {
        this.countryId = countryId;
    }

    public String getCountryName()
    {
        return countryName;
    }
    public void setCountryName(String countryName)
    {
        this.countryName = countryName;
    }

    public int getOrdering()
    {
        return ordering;
    }
    public void setOrdering(int ordering)
    {
        this.ordering = ordering;
    }

    public String getISO3166()
    {
        return iso3166;
    }
    public void setISO3166(String iso3166)
    {
        this.iso3166 = iso3166;
    }
}

File: PHP/Miscellaneous/Country.php
<?php
namespace LTD_API;

class Country
{
    private $countryId;
    private $countryName;
    private $ordering;
    private $iso3166;
    
    public function getCountryId()
    {
        return $this->countryId;
    }
    public function setCountryId($countryId)
    {
        $this->countryId = $countryId;
    }

    public function getCountryName()
    {
        return $this->countryName;
    }
    public function setCountryName($countryName)
    {
        $this->countryName = $countryName;
    }

    public function getOrdering()
    {
        return $this->ordering;
    }
    public function setOrdering($ordering)
    {
        $this->ordering = $ordering;
    }

    public function getIso3166()
    {
        return $this->iso3166;
    }
    public function setIso3166($iso3166)
    {
        $this->iso3166 = $iso3166;
    }
}

